src/kit/Section.module.scss

```diff
@@ -0,0 +1,29 @@
+/* stylelint-disable no-descending-specificity */
+
+.base {
+  &:not(:last-child) {
+    padding-bottom: 16px;
+  }
+  .header {
+    margin-bottom: 8px;
+  }
+  .title {
+    height: 32px;
+    line-height: 32px;
+  }
+  .title.string {
+    font-size: 16px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  & + section.base {
+    border-top: var(--theme-stroke-width) solid var(--theme-stage-border);
+    padding-top: 16px;
+  }
+}
+.base.titleDivider .header {
+  border: 0 solid var(--theme-stage-border);
+  border-bottom-width: var(--theme-stroke-width);
```

Comment:
nit: just add the width to the `border` line instead of defining it as 0 and then the variable